pkgs/tools/security/cmospwd/default.nix

```
@@ -12,6 +12,8 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-8pbSl5eUsKa3JrgK/JLk0FnGXcJhKksJN3wWiDPYYvQ=";
   };
 
+  patches = [ ./fix-compiler.patch ];
```

```suggestion
  makeFlags = [ "CC:=$(CC)" ];
```

There is no need for a patch.